pkg/config/types/auth.go

```diff
@@ -34,3 +37,28 @@ type AuthenticatorConfig struct {
 	Type       string `yaml:"Type,omitempty" json:"Type,omitempty"`
 	PolicyPath string `yaml:"PolicyPath,omitempty" json:"PolicyPath,omitempty"`
 }
+
+type AuthUser struct {
+	Alias        string       `yaml:"Alias,omitempty" json:"Alias,omitempty"`
+	Username     string       `yaml:"Username,omitempty" json:"Username,omitempty"`
+	Password     string       `yaml:"Password,omitempty" json:"Password,omitempty"`
```

Comment:
Are you using a single field for plain and hashed password? That will be nice

Reply:
Correct, we are using the same field for plain passwords and hashed passwords (bcrypt ones)